views/skus/edit.erb

```diff
@@ -0,0 +1,81 @@
+<div class="container">
```

Comment:
This form doesn't seem to be updating the sku. This could be a problem with this file or within the routes, but when I tried to update a sku, nothing changed.